worker/provisioner/provisioner_test.go

```diff
@@ -1539,7 +1536,7 @@ func assertAvailabilityZoneMachines(c *gc.C,
 	}
 	if len(failedAZMachines) > 0 {
 		for _, m := range failedAZMachines {
-			// Is the failed machine listed as failed in AvailabilityZoneMachine
+			// Is the failed machine listed as failed in at least zone?
```

Comment:
at least *one* zone?

Reply:
Depending on now a test is constructed, the machine could be failed in more than one zone.